curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/6555d729c9bfd7c3195f1948.md

```diff
@@ -0,0 +1,755 @@
+---
+id: 6555d729c9bfd7c3195f1948
+title: Step 54
+challengeType: 0
+dashedName: step-54
+---
+
+# --description--
+
+`textContent` is one of the properties of `DOM`. It sets the text content of a node and allows you to set or retrieve the text content of an HTML element.
+
+```html
+<div id="example">This is some text content</div>
+```
+
+```js
+const element = document.getElementById('example');
+console.log(element.textContent); // Output: This is some text content
+
+// Changing the text content
+element.textContent = "Modified text content";
+console.log(element.textContent); // Output: Modified text content
+```
+
+Use a `ternary` operator to check if `currentTitle` is truthy. If so, implicitly return `currentTitle`  otherwise implicitly return an empty string. Assign this result to `playingSong.textContent`.
```

Comment:
@gikf 

For context, this will be the third project in the curriculum and ternary was introduced in project 1 and used in project 2.
So campers will be familiar with it and won't need an explanation or code example. 👍 

But I agree, that the description can be cut down a little bit

Reply:
Thanks, that makes sense.